tests/test-definitions.txt

```diff
@@ -42,6 +42,11 @@ shell_client priority=2500 parallelity=2 single suffix=vst -- --vst true
 shell_client_transaction priority=750 single suffix=http -- $EncryptionAtRest
 shell_client_transaction priority=1750 single suffix=http2 -- --http2 true
 shell_client_transaction priority=2750 single suffix=vst -- --vst true
+shell_client_traffic priority=250 parallelity=1 single suffix=http -- http true
+shell_client_traffic priority=250 parallelity=1 single suffix=http2 -- --http2 true 
+shell_client_traffic priority=250 parallelity=1 single suffix=vst -- --vst true
+shell_client_traffic priority=250 parallelity=1 single suffix=ssl-http -- http true --protocol ssl
+shell_client_traffic priority=250 parallelity=1 single suffix=ssl-http2 -- --http2 true --protocol ssl
```

Comment:
Do we also have VST over SSL?

Reply:
I don't think that the combination of VST over SSL exists.

Reply:
But actually... I may be wrong. Need to check.

Reply:
Added that combination to tests as well, after fixing the testing framework for it.